tests/Controller/CRUDControllerTest.php

```diff
@@ -149,9 +148,6 @@ class CRUDControllerTest extends TestCase
      */
     private $translator;
 
-    /**
-     * @var LoggerInterface&MockObject
-     */
```

Comment:
Not sure about that one…

Reply:
Looks like a bug in the php-cs-fixer version

Reply:
Here are the rules that are applied when I apply everything but this: `no_superfluous_phpdoc_tags, no_empty_phpdoc, no_whitespace_in_blank_line, no_extra_blank_lines`

Reply:
See https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4913

Reply:
Looks like there is currently no way to pin a flintCI container to a specific minor or patch version of php-cs-fixer: https://hub.docker.com/r/flintci/php-cs-fixer/tags